Makefile.in

```diff
@@ -29,7 +29,7 @@
 #
 .POSIX:
 
-VERSION = 2.5.0
+VERSION = 2.5.5
 
```

Comment:
Version `2.5.1` please.

Reply:
Feel free to take care of this if you get around to it before me.